modules/codegen/src/main/java/org/apache/ignite/codegen/MessageCodeGenerator.java

```diff
@@ -683,7 +663,6 @@ else if (Map.class.isAssignableFrom(type)) {
 
     /**
      * @param type Field type.
-     * @param name Field name.
      * @param colItemType Collection item type.
```

Comment:
I don't think we should remove `@param name` documentation here, as field name is still used by the method implementation.